ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_extrema_finding.py

```diff
@@ -344,3 +344,45 @@ def test_numpy_fmax(
         out=None,
         where=where,
     )
+
+
+# fmin
+@handle_frontend_test(
+    fn_tree="numpy.fmin",
+    dtype_and_inputs=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        num_arrays=2,
+        min_value=-np.inf,
+        max_value=np.inf,
+        shared_dtype=True,
+    ),
+    where=np_frontend_helpers.where(),
+    number_positional_args=np_frontend_helpers.get_num_positional_args_ufunc(
+        fn_name="fmin"
+    ),
+)
+def test_numpy_fmin(
+    dtype_and_inputs,
+    where,
+    frontend,
+    test_flags,
+    fn_tree,
+    on_device,
+):
+    input_dtypes, xs = dtype_and_inputs
```

Comment:
Then something resembling the below should be used
`input_dtypes, x, casting, dtype = dtypes_values_casting`